packages/itwin/ec3-widget/src/components/ExportModal.tsx

```diff
@@ -198,7 +198,7 @@ export const ExportModal = (props: ExportProps) => {
 
   return (
     <Modal
-      data-testid="export-modal"
+      data-testid={"ec3-export-modal"}
```

Comment:
In some places you added `data-testid` with curly braces, and in some - without. Make it consistent.

Reply:
went through all the test ids and checked them to be dash separated and without braces